### PR TITLE
Better support for MySQL MULTI_STATEMENTS.

### PR DESCRIPTION
**Pull request by reltuk**

### CODE REVIEW

File: go/vt/sqlparser/ast.go [hunk 1]
```diff
@@ -179,6 +198,20 @@ func ParseStrictDDL(sql string) (Statement, error) {
 	return tokenizer.ParseTree, nil
 }
 
+// ParseOneStrictDDL is the same as ParseOne but it errors on partially parsed
+// DDL statements.
+func ParseOneStrictDDL(sql string) (Statement, int, error) {
```
Comment (zachmu) [inline go/vt/sqlparser/ast.go:203]: Either now or later, let's just get rid of lenient DDL parsing, it's outlived its usefulness

Comment (zachmu) [reply go/vt/sqlparser/ast.go:203]: I guess just file an issue for now

**Review by zachmu — APPROVED** [review]
LGTM, not too gross at all

File: go/vt/sqlparser/ast.go (hunk 1 above)
Comment (reltuk) [reply go/vt/sqlparser/ast.go:203]: #98

**Pull request merged**